tests/engine/test_workflow.py

```diff
@@ -189,6 +189,9 @@ def test_health_settings(self):
             ]
         }
 
+    def test_timeout_seconds(self):
+        assert wf.timeout_seconds == 42.5
```

Comment:
probably should change this to an integer

Reply:
done